packages/esm-ward-app/src/hooks/useInpatientRequest.ts

```diff
@@ -0,0 +1,14 @@
+import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import type { InpatientRequest } from '../types';
+
+//The api is still in implementation stage and this is just a placeholder for the api
```

Comment:
make it a "TODO" comment (some IDEs have special highlighting for them).

Reply:
Made the change. Thanks @chibongho .